odd-platform-api/src/main/resources/db/migration/V0_0_11__add_namespace_support.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE data_source
+    ADD COLUMN namespace_id BIGINT REFERENCES namespace (id);
+
+ALTER TABLE data_entity
```

Comment:
Don't we get concerned about back compatibility issues in case of a forced rollback? Let's say someone upgrades the app (and therefore, the database) to a newer version, but something goes wrong and they're forced to rollback to previous version, which, in turn, won't have this field available and something might break. Usually this can be solved by dropping the column in a next app version.

Reply:
We definitely are going to leverage that practice in the future, but right now it's not necessary since we didn't have namespace API until this PR. All of the namespace_id values are null